applications/GeoMechanicsApplication/tests/cpp_tests/custom_utilities/test_stress_strain_utilities.cpp

```diff
@@ -109,6 +118,21 @@ KRATOS_TEST_CASE_IN_SUITE(CheckCalculateMohrCoulombShearCapacityShearOnly, Krato
                                      stress_vector, cohesion, friction_angle));
 }
 
+KRATOS_TEST_CASE_IN_SUITE(CalculateMohrCoulombShearCapacityThrowsWhenPhiIsOutOfBounds, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    Vector stress_vector(4);
+    stress_vector <<= -2.0, 0.0, 2.0, 0.0;
+    constexpr auto cohesion                  = 2.0;
+    auto           friction_angle_in_radians = -MathUtils<>::DegreesToRadians(0.0001);
```

Comment:
Perhaps this is slightly clearer:
```suggestion
    auto           friction_angle_in_radians = MathUtils<>::DegreesToRadians(-0.0001);
```

Reply:
changed